pkg/eval/builtin_special.go

```diff
@@ -359,22 +366,43 @@ func use(fm *Frame, spec string, r diag.Ranger) (*Ns, error) {
 	if libDir == "" {
 		return nil, noSuchModule{spec}
 	}
-	return useFromFile(fm, spec, libDir+"/"+spec+".elv", r)
+	var path string
+	_, plugin := os.Stat(libDir + "/" + spec + ".so")
+	if os.IsNotExist(plugin) {
+		path = libDir + "/" + spec + ".elv"
+	} else {
+		path = libDir + "/" + spec + ".so"
+	}
+	return useFromFile(fm, spec, path, r)
```

Comment:
This is duplicate with the code above.

Since both places call `useFromFile` and these are the only call sites of `useFromFile`, one could change the semantics of the `path` argument to not include the extension, and let `useFromFile` determine whether to use the `.so` file or the `.elv` file. It'd also eliminate the need for the `strings.HasSuffix` check in `useFromFile`.